src/controllers/TaskController.ts

```diff
@@ -1,14 +1,27 @@
 import { Request, Response } from "express";
 import { Task, TaskModel  } from "../models/Task";
 import {TaskService} from "../services/TaskService";
+import { Error } from "mongoose";
+import { formatErrors } from "../database/formatErrors";
 
 let taskService = new TaskService();
 
 export const createTask = async (req: Request, res: Response) => {
+    let errors: {[key: string]: string} | null = null;
     const requestBody = await req.body;
-    await taskService.createTask(requestBody);
+    const taskRes: Error.ValidationError | Task | null = await taskService.createTask(requestBody);
+    if(taskRes) {
+        if (taskRes instanceof Error.ValidationError) {
+            errors = formatErrors(taskRes);
+        }
 
-    res.status(200).json({"Message":"Task created!"});
+        if(errors) {
+            res.status(400).json(errors);
```

Comment:
usually for errors use 500 which would be Internal Server Error code, just some small thing I noticed